errors/types.go

```diff
@@ -19,8 +19,8 @@ import (
 )
 
```

Comment:
ignore this file, it will disappear once previous https://github.com/ossf/scorecard/pull/712 and https://github.com/ossf/scorecard/pull/718 are merged